tests/Common/TriggersTest.php

```diff
@@ -344,24 +364,41 @@ public function testUpdateTriggerWithNonMasterToken(TokenCreateOptions $options)
         $this->assertEquals([['tableId' => 'in.c-API-tests.watched-1']], $updatedTrigger['tables']);
 
         // try it even with non-master token but this token didnt create this trigger
```

Comment:
```suggestion
        // try to update trigger with non-master without permissions
```

Reply:
Lepší by bylo aby ty proměnné byly samopopisné :) `$clientWithTokenWithoutPermissions`

Reply:
To "another", ještě 2x, je tam zbytečné IMHO